src/index.ts

```diff
@@ -1,8 +1,164 @@
+import testData from './testData';
+import { EventsPhase } from './types/Phases';
+import {
+  Event,
+  DurationEvent,
+  CompleteEvent,
+  MetadataEvent,
+  SampleEvent,
+  ObjectEvent,
+  ClockSyncEvent,
+  AsyncEvent,
+  ContextEvent,
+  InstantEvent,
+  CounterEvent,
+  FlowEvent,
+  MemoryDumpEvent,
+  MarkEvent,
+  LinkedIDEvent,
+} from './types/EventInterfaces';
 export const sum = (a: number, b: number) => {
   if ('development' === process.env.NODE_ENV) {
     console.log('boop');
   }
   return a + b;
 };
+const events: Event[] = testData.traceEvents;
```

Comment:
To get the build to pass until we fix the string enum types, you can do this
```suggestion
const events: Event[] = testData.traceEvents as Event[];
```

It's not a good idea in general as it can hide other type incompatibilities, but for now is necessary if you want the build to go green before merging